src/plugins/intel_gpu/include/intel_gpu/plugin/graph.hpp

```diff
@@ -83,14 +83,16 @@ class Graph {
     bool use_external_queue() const;
 
 protected:
+    InferenceEngine::gpu::ClContext::Ptr m_context;
+    std::shared_ptr<Program> m_program;
+    std::string m_networkName;
+    Config m_config;
+    uint16_t m_stream_id;
```

Comment:
What is the motivation of this change? will the graph is created per stream???

Reply:
Ah.. they were just moved..

Reply:
You can look at the last commit only here. Most of the other changes are covered here: https://github.com/openvinotoolkit/openvino/pull/12172